product_docs/docs/migration_toolkit/55/07_invoking_mtk/08_mtk_command_options.mdx

```diff
@@ -603,6 +725,14 @@ Include the `-allSynonyms` option to migrate all public and private synonyms fro
 
 Include the `-allPublicSynonyms` option to migrate all public synonyms from an Oracle database to an EDB Postgres Advanced Server database. If a synonym with the same name already exists in the target database, the existing synonym is replaced with the migrated version.
 
+`-excludeSynonyms <synonym_list>` 
+
+Exclude selected synonyms from migration. 
+The `synonym_list` is a comma-separated list of synonym names with no intervening space characters.
+This option applies when all synonyms from a schema are selected for migration using the `-allSynonyms` option.
```

Comment:
@zahid75 Does the `-excludeSynonyms` option not also get applied by default if `-allSynonyms` is not specified at all? 

Also, what is the behavior if `-excludeSynonyms` is used with `-allPublicSynonyms` or the `-allPrivateSynonyms` options? Some text should be added here to identify whether that is supported or not and what the behavior is if `-excludeSynonyms` is used with either of these options.

Reply:
Explained above.